docs/List.md

```diff
@@ -0,0 +1,466 @@
+      <Carousel.Tile>
+        <Statistic
+          title='Total Cost (AUD)'
+          prefix='AUD'
+          value='54,118.00'
+        />
+      </Carousel.Tile>
+      <Carousel.Tile>
+        <Statistic
+          title='Total Cost (DZD)'
+          prefix='DZD'
+          value='99.00'
+        />
+      </Carousel.Tile>
+    </Carousel>
+</>
+```
+
+
+### Pagination section
+All list should have [Pagination](/#/Interaction/Pagination) when the page is customer facing
+It should be hidden when there is no record (also no result when filtering)
+
+```jsx
+import { PaginationDetails, PaginationButtons } from '@Domain/Paginators';
+import { GridLayout } from '@Domain/Layouts';
+import { Variables } from '@Common';
+  
+  initialState = { page: 1 };
+  
+  const props = { totalPages: 11, totalCount: 105, pageSize: 10};
+  
+  <GridLayout
+      gutterMarginY={Variables.Spacing.sXSmall}
+      horizontalAlignment={GridLayout.HorizontalAlignment.Justify}
+      verticalAlignment={GridLayout.VerticalAlignment.Middle}
+    cells={[
+            {
+               size: { desktop: 6, min: 12 },
+               content: <PaginationDetails
+                          currentPage={state.page}
+                          totalPages={props.totalPages}
+                          totalCount={props.totalCount}
+                          pageSize={props.pageSize}
+                        />,
+               displayType: 'flex',
+               flexHorizontalAlignment: { desktop: GridLayout.HorizontalAlignment.Left, min: GridLayout.HorizontalAlignment.Center }
+             },
+             {
+               size: { desktop: 6, min: 12 },
+               content: <PaginationButtons
+                          currentPage={state.page}
+                          totalPages={props.totalPages}
+                          handlePageChange={(page) => setState({page})}
+                        />,
+               displayType: 'flex',
+               flexHorizontalAlignment: { desktop: GridLayout.HorizontalAlignment.Right, min: GridLayout.HorizontalAlignment.Center }
+             }
+           ]}
+  />
+```
+
+### Loading state
+List section (from top to bottom): filter and sort section, statistics tile, cards, pagination with sLarge spacing 24px in between
+
+Filter and sort section goes together as a 40px height skeleton
+Statistics tile would depend on the number of tile would the list have (It would be depend on minimum number of tile so it would not include the optional ones)
+The skeleton would be 200 x 90 with 8 margin in between Cards section skeleton would have three cards only and the skeleton height would depend on the card height on that page
+
+Pagination section would not have a skeleton for it
+
+```jsx
+import { BlockSkeleton } from '@Domain/Skeletons';
+import { Inline } from '@Domain/Layouts';
+import { Variables } from '@Common';
+
+<>
+  <BlockSkeleton margins={{ bottom: Variables.Spacing.sLarge}} height={Variables.Spacing.s2XLarge}/>
+  <Inline
```

Comment:
Should this also be in a carousel? not sure if that works with inline though.

Reply:
inline is simpler and just want to have 8 px gap

Reply:
i mean the inline within a carousel. or else mobile skeletons might look weird.